tests/layer_tests/tensorflow_tests/test_tf_BinaryOps.py

```diff
@@ -39,19 +39,16 @@ def _prepare_input(self, inputs_dict):
     def create_add_placeholder_const_net(self, x_shape, y_shape, ir_version, op_type,
```

Comment:
it's better to change the name of this method

Reply:
It is out of scope of this PR. In this PR I only removed useless and confusing comments.

Reply:
it's also confusing a bit, we can change it in the next pr